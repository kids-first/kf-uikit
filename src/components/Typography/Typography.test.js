import React from 'react';
import { H1, H2 } from '.';
import { createSerializer } from 'jest-emotion';
import * as emotion from 'emotion';
import styled from 'react-emotion';

// it('should render with default styles', () => {
//   const actual = create(<H2>Heading H2</H2>);
//   expect(actual).toMatchSnapshot();
// });

// expect.addSnapshotSerializer(createSerializer(emotion));

test('renders with correct styles', () => {
  // const H1 = styled('h1')`
  //   float: left;
  //   ${({ theme }) => theme.h1u};
  // `;
  const tree = create(<H1>hello world</H1>);

  expect(tree).toMatchSnapshot();
});

test('renders with correct styles for H2', () => {
  // const H1 = styled('h1')`
  //   float: left;
  //   ${({ theme }) => theme.h1u};
  // `;
  const tree = create(<H2>hello world</H2>);

  expect(tree).toMatchSnapshot();
});
