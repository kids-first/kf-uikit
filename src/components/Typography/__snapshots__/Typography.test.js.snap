// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with correct styles 1`] = `
.circuit-0 {
  font-family: Montserrat,Helvetica,sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: none;
  font-size: 36px;
  font-weight: 500;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  background-image: linear-gradient(to right,#2b388f,#009bb8 51%,#02b0ed), linear-gradient(#2b388f,#2b388f);
  background-clip: text;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
}

<h1
  className="circuit-0"
>
  hello world
</h1>
`;

exports[`renders with correct styles for H2 1`] = `
.circuit-0 {
  font-family: Montserrat,Helvetica,sans-serif;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: none;
  color: #2b388f;
  font-size: 30px;
  line-height: 0.95;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
  color: #2b388f;
  font-weight: 500;
}

<h2
  className="circuit-0"
>
  hello world
</h2>
`;
